Added a new option in the Table Panel to change the size of the cells padding.

LucasArona opened the pull request:
fixes #9802

LucasArona committed:
Added a new option in the Table Panel to change the size of the cells padding.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=9803) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=9803) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=9803) it.</sub>

LucasArona committed:
fixed 2 warning:
  71:7  warning  Declarations should come before nestings  declarations-before-nesting
  72:7  warning  Declarations should come before nestings  declarations

torkelo commented:
thanks for contributing! We are a bit hesitant to add more options before knowing if it's going to be generally useful, so closing this PR for now until we can see more demand for the feature request
https://github.com/grafana/grafana/issues/9802

lperrin-tower commented:
can you please revisit the decision to merge this? it would actually be really helpful to pack more info in the tables. thanks
